### PR TITLE
[FE-13] User Interface for Surveys

### PR DESCRIPTION
**Pull request by GabrielaY**
Added all front end functions
- show survey
- vote on survey
- close survey
- elect winner
- create survey
- error handling

### CODE REVIEW

File: odyssey/client/src/Profiles/MyProfile.svelte [hunk 1]
```diff
@@ -189,6 +227,26 @@
...
 		{/each}
+		{#each surveys as survey}
+			
+			<div class='post-box'>
+
+				<h3>{survey.text}</h3>
+
+				<img class="post-image" src={"/images/" + user.username + "/" + survey.image_path}>
+
+				{#if survey.is_open == true}
```
Comment (ogi02) [inline odyssey/client/src/Profiles/MyProfile.svelte:238]: User should be able to see the current results of the survey, not only the Close button

File: odyssey/client/src/Profiles/UserProfile.svelte [hunk 2]
```diff
@@ -42,20 +47,22 @@
...
+		follow_result = {
 			profile_name: user.username
 		};
 
-		is_following = await isFollowing(result);
+		is_following = await isFollowing(follow_result);
 
 		await loadMorePosts();
+		await loadMoreSurveys();
+		console.log(surveys);
```
Comment (ogi02) [inline odyssey/client/src/Profiles/UserProfile.svelte:64]: Please remove console logs

Comment (GabrielaY) [reply odyssey/client/src/Profiles/UserProfile.svelte]: Please review the new changes everything should be fine now

File: odyssey/client/src/Profiles/MyProfile.svelte (hunk 1 above)
Comment (ogi02) [reply odyssey/client/src/Profiles/MyProfile.svelte:249]: Checked changes. Well done ❤️

File: odyssey/client/src/Profiles/UserProfile.svelte (hunk 2 above)
Comment (ogi02) [reply odyssey/client/src/Profiles/UserProfile.svelte]: Perfect! 👌

File: odyssey/client/src/Profiles/MyProfile.svelte (hunk 1 above)
Comment (valentin30) [reply odyssey/client/src/Profiles/MyProfile.svelte:249]: you can remove == true - https://svelte.dev/tutorial/if-blocks =))

Comment (ogi02) [reply odyssey/client/src/Profiles/MyProfile.svelte:249]: The issue is addressed in [this pull request](https://github.com/ogi02/Odyssey/pull/43). Thanks for bringing it up 😀

**Pull request merged**